developer_manual/general/devenv.rst

```diff
@@ -18,29 +18,29 @@ To be able to use and develop with ownCloud takes a few steps.
 If you have already completed one or more of these steps, feel free to skip them.
 Otherwise, if you're just getting started, begin by getting the ownCloud source code.
 
-Install the Core Software 
+1. Install the Core Software 
 -------------------------
 
 The first thing to do is to ensure that your server has the necessary software for installing and running ownCloud.
-While you can go further, you need to install at least the following:
 
-- `Make <https://www.gnu.org/software/make/>`_
-- `Node.js <https://nodejs.org>`_.
-- `git <https://git-scm.com/>`_
-- `npm <https://www.npmjs.com/>`_
-- `unzip <https://linux.die.net/man/1/unzip>`_
-- `wget <https://www.gnu.org/software/wget/>`_
+https://doc.owncloud.com/server/latest/admin_manual/installation/source_installation.html#install-the-required-packages
 
-Install Dependencies on Ubuntu 16.04
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Then you will need to install software required to run the installation process for the development environment.
```

Comment:
Then you will need to install the software required to run the development environment's installation process.